files/ko/learn/css/building_blocks/backgrounds_and_borders/index.md

```diff
@@ -109,21 +185,43 @@ CSS {{cssxref("background")}} 속성은 이 수업에서 만나게 될 많은 ba
 - 길이 단위를 제거하고 `background-size: cover` 또는 `background-size: contain` 을 사용할 때 어떻게 되는지 확인 하십시오.
 - 이미지가 박스보다 작으면, `background-repeat` 의 값을 변경하여 이미지를 반복할 수 있습니다.
 
-{{EmbedGHLiveSample("css-examples/learn/backgrounds-borders/size.html", '100%', 800)}}
+```html live-sample___size
+<div class="box"></div>
+```
+
+```css hidden live-sample___size
+.box {
+  width: 500px;
+  height: 100px;
+  padding: 0.5em;
+  border: 1px solid #ccc;
+  margin: 10px;
+}
+```
+
+```css live-sample___size
+.box {
+  background-image: url(https://mdn.github.io/shared-assets/images/examples/balloons.jpg);
+  background-repeat: no-repeat;
+  background-size: 80px 10em;
+}
+```
+
+{{EmbedLiveSample("size")}}
 
 #### 배경 이미지 배치
 
 {{cssxref("background-position")}} 속성을 사용하면 적용되는 박스에서 배경 이미지가 나타나는 위치를 선택할 수 있습니다. 박스의 왼쪽 상단 모서리가 `(0,0)` 이고, 박스가 가로 (`x`) 및 세로 (`y`) 축을 따라 위치하는 좌표계를 사용합니다.
 
-> **참고:** `background-position` 의 기본값은 `(0,0)` 입니다.
+> [!NOTE] `background-position` 의 기본값은 `(0,0)` 입니다.
```

Comment:
Just in case you have formatting problems, if the first character in the note is a special character like backticks, I've had success elsewhere doing:

````markdown
> [!NOTE]
> The `background-position` prop is
````

instead of

````markdown
> [!NOTE] `background-position` is
````

Reply:
@hochan222 님 이 커멘트 확인해주실 수 있으실까요?

Reply:
@bsmth, Sorry for the delay in getting back to you.

Even without line breaks, it seems to convert well, but I’m curious if there might be any issues. When translating into Korean, it's difficult to position some words before the backtick, so adding an emoji like 📎 seems to be the only option.

```
> [!NOTE]
> 📎 `background-position` 의 기본값은 `(0,0)` 입니다.
```

Reply:
No problem. I think there was an issue with Prettier formatting it badly in EN-US if the first character after `> [!NOTE]` is **not alphanumeric**, but if that's working here, then all is well, you don't have to include additional characters etc. 👍🏻